### PR TITLE
OCPBUGS-31423: Try to pull installer image before installing

### PR DESCRIPTION
**Pull request by CrystalChun**
https://issues.redhat.com/browse/OCPBUGS-31423
Split the install command into two parts: pulling the installer image then running the install command. Pulling separately and ahead of time prevents possible flakes when pulling the image. Mostly relevant for day 2 hosts since we don't run the container image availability command before installing.

/cc @tsorya

### CODE REVIEW

File: src/commands/actions/install_cmd.go [hunk 1]
```diff
@@ -276,3 +288,29 @@ func (a *install) Command() string {
...
+	var err error
+	for attempts := 0; attempts < retry; attempts++ {
+		if err = pullImage(pullTimeoutSeconds, image); err == nil {
+			break
+		}
+	}
+	return err
+}
+
+func pullImage(pullTimeoutSeconds int64, image string) error {
```
Comment (tsorya) [inline src/commands/actions/install_cmd.go:302]: Lets validate if image exists beforehand, another way we will pull it all the time

Comment (CrystalChun) [reply src/commands/actions/install_cmd.go:319]: Good idea! Added it!

**Comment by CrystalChun** [comment]
Oh @tsorya may I get your approval? 😄

**Pull request merged**